elementary/config/config.py

```diff
@@ -121,6 +123,18 @@ def __init__(
             GroupingType.BY_ALERT.value,
         )
 
+        self.provided_slack_in_cli = slack_webhook is not None or (
```

Comment:
I understand your point, however the intended behavior is that if you pass something to the command line it should take precedence over what you have in `~edr.config.yml`. So in my opinion if you pass one webhook to the CLI and in your config you have a slack webhook the command should run just fine. In the proposed solution it raises an error I think.

Might be that we need to rewrite it a bit, but don't see any other way for now. If you have a suggestion please let me know.

Reply:
@IDoneShaveIt , think it could be argued that you just should not supply multiple types of integration methods. In that case you also don't have the issue with precedence of the CLI over the config file.

Could you let me know if you agree and then I will apply you suggestions?

Reply:
@FrankTub I think is fair enough to assume and guide the users to use config / CLI args and not both.
There is some work that could be done around the connection between the two regardless this contribution.
For now I think that leaving things as simple as possible is the way to go, and if someone will provide 2 integrations (one from the config and the other from the CLI args) he will just get an error 🙂 

So I think we can apply the suggestions for now 🙂

Reply:
I implemented the suggestions. Let me know what you think of the `Config` class in this way!